nautobot/nautobot_data_validation_engine/urls.py

```diff
@@ -0,0 +1,85 @@
+"""Django urlpatterns declaration for nautobot_data_validation_engine app."""
+
+from django.urls import path
+
+from nautobot.core.views.routers import NautobotUIViewSetRouter
+from nautobot.extras.views import ObjectChangeLogView, ObjectNotesView
+from nautobot.nautobot_data_validation_engine import models, views
+
+app_name = "nautobot_data_validation_engine"
+
+router = NautobotUIViewSetRouter()
+router.register("data-compliance", views.DataComplianceListView)
+router.register("regex-rules", views.RegularExpressionValidationRuleUIViewSet)
+router.register("min-max-rules", views.MinMaxValidationRuleUIViewSet)
+router.register("required-rules", views.RequiredValidationRuleUIViewSet)
+router.register("unique-rules", views.UniqueValidationRuleUIViewSet)
+
+urlpatterns = [
+    path(
+        "data-compliance/<uuid:pk>/changelog/",
+        ObjectChangeLogView.as_view(),
+        name="datacompliance_changelog",
+        kwargs={"model": models.DataCompliance},
+    ),
+    path(
+        "data-compliance/<uuid:pk>/notes/",
+        ObjectNotesView.as_view(),
+        name="datacompliance_notes",
+        kwargs={"model": models.DataCompliance},
+    ),
```

Comment:
Changelog/notes patterns should be removable now as auto-provided by use of NautobotUIViewSet.